contrib/sb-aclrepl/README

```diff
@@ -16,12 +16,13 @@ EXAMPLE ~/.sbclrc FILE
 ======================
 
 (ignore-errors (require 'sb-aclrepl))
-
```

Comment:
debugger invoked on a SB-C:COMPILER-ERROR in thread
#<THREAD "main thread" RUNNING {1000510083}>:
  READ error during LOAD:

    Package ASDF does not exist.

      Line: 10, Column: 36, File-Position: 231

      Stream: #<SB-SYS:FD-STREAM for "file /home/lin/.sbclrc" {1001526F23}>

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [CONTINUE    ] Replace form with call to ERROR.
  1: [SIGNAL-ERROR] SB-C::SIGNAL-ERROR
  2: [ABORT       ] Abort userinit file "/home/lin/.sbclrc".
  3:                Skip to toplevel READ/EVAL/PRINT loop.
  4: [EXIT        ] Exit SBCL (calling #'EXIT, killing the process).

(SB-C:COMPILER-ERROR SB-C::INPUT-ERROR-IN-LOAD :CONDITION #<SB-INT:SIMPLE-READER-PACKAGE-ERROR "Package ~A does not exist." {1001694B43}> :STREAM #<SB-SYS:FD-STREAM for "file /home/lin/.sbclrc" {1001526F23}>)